src/main/resources/templates/home.html

```diff
@@ -2,58 +2,61 @@
 <html lang="en">
 
 <head>
-  <title>Green Goblins</title>
-  <link rel="stylesheet" href="css/stylesheet.css">
+  <title>Goblin Tees | Home</title>
+  <link rel="stylesheet" href="css/main-style.css">
+  <link rel="stylesheet" href="css/home-style.css">
```

Comment:
do we want both stylesheets here?

Reply:
Yeah, I want to separate the styling we're going to reuse on all pages i.e nav, header, etc on main-style.
And create separate styles sheets for each page.

If it's overkill I can make changes.